yarn.lock

```diff
@@ -10673,6 +10673,14 @@ react-style-singleton@^2.2.1:
     invariant "^2.2.4"
     tslib "^2.0.0"
 
+react-style-singleton@^2.2.3:
```

Comment:
singleton got duplicated, but this is not _really_ affecting bundle size. I'll take a look

Reply:
I have just `pnpm install` this one.